testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/Environment.java

```diff
@@ -553,6 +593,51 @@ private Environment build(Optional<EnvironmentListener> listener)
             return outputFrame -> {};
         }
 
+        private void enableConfiguredConnectorsTest()
+        {
+            if (!containers.containsKey(TESTS)) {
+                return;
+            }
+            DockerContainer container = containers.get(TESTS);
```

Comment:
nit: rename to testContainer and use below in String[] commandParts = testContainer.getCommandParts()

Reply:
Done, missed it when extracting a method, thanks!